recipes/packio/all/test_package/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ if (PACKIO_VERSION VERSION_LESS "1.2.0")
 elseif (PACKIO_VERSION VERSION_LESS "2.0.0")
     add_executable(main 1.2.x.cpp)
 else ()
-    add_executable(main 2.0.x.cpp)
+    add_executable(main 2.0.x-2.1.x.cpp)
```

Comment:
Can you please rename to test_package.cpp or example.cpp ? This way it does not need to change in the near future.

Reply:
Maybe "latest.cpp" ?

Reply:
My suggestion is based on the existing ones. 

It's not a blocker but it's odd to change this for a bump

Reply:
I agree it should not change in the near future.
I went for "latest.cpp", it makes more sense to me considering there are 1.2.x.cpp and 1.0.x-1.1.x.cpp in the same directory.